app/assets/javascripts/application.js

```diff
@@ -0,0 +1,48 @@
+// Load all the things
```

Comment:
Instead of this could we add a comment explaining that this uses gulp-include so there is some explanation for this formatting (which doesn't appear anywhere else in the app and is not part of native JS)?